Remotion/ObjectBinding/Web/Res/Themes/NovaGray/Html/BocList.css

```diff
@@ -361,6 +361,13 @@ th.bocListTitleCell > a[href]:active
   background-color: #C5E0FC;
 }
 
+th.bocListTitleCell > a > img,
+th.bocListTitleCell > span > img
+{
+  width: 16px;
+  height: 16px;
+}
```

Comment:
I think we shoudl also add a padding to the follow-up span, like in NovaViso.

Reply:
The reason I did not introduce it in NovaGray is that the icons there don't have a spacing (wrong icon in column to illustrate):
![image](https://github.com/re-motion/Framework/assets/5082673/eb9f4999-62ad-4e9e-a32f-79b3228c830b)

I changed the icon from 12x17 to 16x16 and used it here with a small shift upwards, leaving us with this:
![image](https://github.com/re-motion/Framework/assets/5082673/e6ad4f1c-e821-4da8-a1f1-0175a3cea30e)

Because the icon has some whitespace left and right we don't need to introduce additional margin, which fits in with the existing formatting.